index.html

```diff
@@ -60,7 +60,7 @@
 			Your slides don't have to be this way, and it doesn't take a whole lot of effort to make them better.
 		</p>
 		<p>
-			PowerPointLabs makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
+			PowerPointLabs is an open source project based in the National University of Singapore. It makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
```

Comment:
Bold `open source project` and `National University of Singapore`? Can link each too (first one to github and second one to SoC website)